packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,82 @@
 ---
 title: Okta API Products release notes 2021
 ---
+## December
+
+### Monthly release 2021.12.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.14.0](#okta-sign-in-widget-version-5-14-0) | December 8, 2021 |
+| [Browser-like devices can now use the Device Authorization Grant flow](#browser-like-devices-can-now-use-the-device-authorization-grant-flow) | December 8, 2021 |
+| [Custom domains with Okta-managed certificates](#custom-domains-with-okta-managed-certificates) | December 8, 2021 |
+| [Device Authorization grant type is now GA in Production](#device-authorization-grant-type-is-now-ga-in-production) | November 3, 2021 |
```

Comment:
```suggestion
| [Device Authorization grant type is now GA in Production](#device-authorization-grant-type-is-now-ga-in-production) | December 8, 2021 |
```

Reply:
@christinechoy-okta - see my comment on the same blurb in the Classic RNs.

Reply:
👍